automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/runtimes/RuntimeCreationDiskSpec.scala

```diff
@@ -58,6 +58,8 @@ class RuntimeCreationDiskSpec
       for {
         getRuntimeResponse <- LeonardoApiClient.createRuntimeWithWait(googleProject, runtimeName, createRuntimeRequest)
         clusterCopy = ClusterCopy.fromGetRuntimeResponseCopy(getRuntimeResponse)
+        rat <- Ron.authToken()
+        implicit0(authToken: AuthToken) = rat
```

Comment:
you can do `implicit0(authToken: AuthToken) <- Ron.authRoken()`

Reply:
Awesome! Pushed a commit changing all occurrences accordingly.